classObjectLesson/com/company/professions/Driver.java

```diff
@@ -0,0 +1,41 @@
+package Rep.classObjectLesson.com.company.professions;
+
+import Rep.classObjectLesson.Person;
+
+public class Driver extends Person {
+    String fullName;
+    int experience;
+
+    public Driver(String personName, int personAge, boolean isMarried, String fullName, int experience) {
```

Comment:
Person here is NOT the same Person as in previous homework
This Person should contain two fields(fullName and age) and toString, equals, hashcode methods